### ISSUE TITLE
NotFoundError: The object can not be found here.

### ISSUE THREAD

**Issue by Qoraiche** [godka/kurento-rtmp#11]
I get this error in the console log

`NotFoundError: The object can not be found here.`

**Comment by godka** [comment]
Hi, thx for your interest. 
Do you have detailed console logs?

**Comment by Qoraiche** [comment]
Yes in your website https://1029.mythkast.net/ console logs, streaming not started and the error I've mentioned before trigger.

Note: i use an old Browser
